playbook/app/pb_kits/playbook/pb_caption/docs/_caption_example.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/jsx-indent-props */
```

Comment:
Why is this override added here? Is it necessary? Generally, as a rule of thumb, we *try very hard* not to disable any lint rules if possible.

Reply:
fixed!  I was testing the lint and forgot to remove